pex/pex_info.py

```diff
@@ -30,7 +30,7 @@ class PexInfo(object):
   requirements: list                # list of requirements for this environment
 
   # Environment options
-  pex_root: ~/.pex                   # root of all pex-related files
+  pex_root: eg: ~/.pex               # root of all pex-related files
```

Comment:
shouldn't this actually include the type information per the format of the other lines, i.e. 'string'? maybe something like:

```pex_root: string, default: ~/.pex                              # root of all pex-related files```